### DISCUSSION TITLE
Beta versions?

### DISCUSSION THREAD

**Discussion by steinhh** [interactive-data-language/vscode-idl#56] [General]
Hi!

Is there a beta version "stream" of the extension that we could follow? We'd be happy to be guinea pigs :)

Cheers,
Stein

**Comment by znorman-harris** [comment]
That's a great question! I did a quick google search and it looks like there's some support for this. I would be happy to try publishing a version to see if it pops up like we expect?

Found this link for VSCode: https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions

This might work for one of Martin's issues to see if fixes in my environment apply to yours as well.
